### ISSUE TITLE
RTX 2060 12 GBVRAM problem loading the model probably

### ISSUE THREAD

**Issue by agustincaniglia** [kijai/ComfyUI-HunyuanVideoWrapper#146]
This is what I get when reaching the model node.

To see the GUI go to: http://127.0.0.1:8188
FETCH DATA from: D:\COMFY_UI_CASA\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
got prompt
Loading text encoder model (clipL) from: D:\COMFY_UI_CASA\ComfyUI_windows_portable\ComfyUI\models\clip\clip-vit-large-patch14
Text encoder to dtype: torch.float16
Loading tokenizer (clipL) from: D:\COMFY_UI_CASA\ComfyUI_windows_portable\ComfyUI\models\clip\clip-vit-large-patch14
Loading text encoder model (llm) from: D:\COMFY_UI_CASA\ComfyUI_windows_portable\ComfyUI\models\LLM\llava-llama-3-8b-text-encoder-tokenizer
Loading checkpoint shards: 100%|█████████████████████████████████████████████████████████| 4/4 [00:48<00:00, 12.10s/it]
Text encoder to dtype: torch.bfloat16
Loading tokenizer (llm) from: D:\COMFY_UI_CASA\ComfyUI_windows_portable\ComfyUI\models\LLM\llava-llama-3-8b-text-encoder-tokenizer
llm prompt attention_mask shape: torch.Size([1, 161]), masked tokens: 12
clipL prompt attention_mask shape: torch.Size([1, 77]), masked tokens: 13
model_type FLOW
Scheduler config: FrozenDict([('num_train_timesteps', 1000), ('shift', 9.0), ('reverse', True), ('solver', 'euler'), ('n_tokens', None), ('_use_default_values', ['num_train_timesteps', 'n_tokens'])])
Using accelerate to load and assign model weights to device...
Requested to load HyVideoModel
loaded completely 9737.65707092285 9737.224731445312 False
Presione una tecla para continuar

I don't know what could it be.

**Comment by kijai** [comment]
If it just crashes, probably running out of RAM.Using bnb_nf4 quantization might help, which requires installing bitsandbytes.

**Comment by raultresd** [comment]
I got 20GB VRAM and I was getting OOM when loading LLama. I solved the problem updating the dependencies, which only updated torch to 2.5.1, it seems to be important to have that torch version.
![image](https://github.com/user-attachments/assets/b4e74292-32ec-41cb-a48e-ca6aebcfe0a7)

**Comment by agustincaniglia** [comment]
Yes I had very little page file I added more and it went through, but then I got OOM because of VRAM and that's about it. Next ear probably buy a 24 GB VRAM card.
